libs/designer-ui/src/lib/peek/index.tsx

```diff
@@ -5,16 +5,18 @@ import { useIntl } from 'react-intl';
 export interface PeekProps {
   input: string;
```

Comment:
instead of using the peek UI component can we use the code component libs\designer-ui\src\lib\code\index.tsx, peek should only ever be readonly

Reply:
Sure thing, I wasn't aware of it. Thank you so much!